chapter2/tests/t1/chapter2-t1-two-files.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#
+# Task 1 - existance of files
+set -eo pipefail
+
+
+# Function for delimiter
+print_separator() {
+  printf '%*s\n' "${COLUMNS:-$(tput cols)}" '' | tr ' ' -
+}
+
+
+# Function to generate a random string and write to a file with permissions
+create_file() {
+  local file="$1"
+  openssl rand -base64 20 > "$file" && chmod 700 "$file"
+}
+
+
+# checking permissions
+if [[ ! -r $(pwd) ]]; then
+  echo 'Directory is not readable.'
+  exit 255
+fi
+
+
+# Args check
+if [[ "$#" -ne 2 ]]; then
+  echo 'Error: Two arguments are required'
+  exit 1
+fi
+
+
+# If the file exists and is a regular file - show, if not - create and fill.
+files=$@
+for file in $files; do
```

Comment:
Something isn't working here.
What if I want files named with spaces (cause Linux allows this):
```
ginto in on ginto-pc on ☸ teleport.saritasa.rocks-teleport.saritasa.rocks (plumbing) ~/TEMP/camp took 29s
➜ ./chapter2-t1-two-files.sh 'my file' 'my second file'
my not exist, it will be created.
file not exist, it will be created.
------------------------------------------------------------------------------------------------------------------------
my is here:
7XIXNixjNETNDAPNCljyUxWQR18=
------------------------------------------------------------------------------------------------------------------------
second not exist, it will be created.
------------------------------------------------------------------------------------------------------------------------
file is here:
P80DhEJhqCnRorwPAJc7EI0iPow=
------------------------------------------------------------------------------------------------------------------------

ginto in on ginto-pc on ☸ teleport.saritasa.rocks-teleport.saritasa.rocks (plumbing) ~/TEMP/camp
➜ ls
chapter2-t1-two-files.sh  file  my  second
```

Reply:
> Something isn't working here. What if I want files named with spaces (cause Linux allows this):
> 
> ```
> ginto in on ginto-pc on ☸ teleport.saritasa.rocks-teleport.saritasa.rocks (plumbing) ~/TEMP/camp took 29s
> ➜ ./chapter2-t1-two-files.sh 'my file' 'my second file'
> my not exist, it will be created.
> file not exist, it will be created.
> ------------------------------------------------------------------------------------------------------------------------
> my is here:
> 7XIXNixjNETNDAPNCljyUxWQR18=
> ------------------------------------------------------------------------------------------------------------------------
> second not exist, it will be created.
> ------------------------------------------------------------------------------------------------------------------------
> file is here:
> P80DhEJhqCnRorwPAJc7EI0iPow=
> ------------------------------------------------------------------------------------------------------------------------
> 
> ginto in on ginto-pc on ☸ teleport.saritasa.rocks-teleport.saritasa.rocks (plumbing) ~/TEMP/camp
> ➜ ls
> chapter2-t1-two-files.sh  file  my  second
> ```

Fixed